.storybook/custom/custom.scss

```diff
@@ -603,31 +603,37 @@ body.sb-show-main {
   .card-one-dimensions {
     width: 40rem;
     height: 16rem;
+    margin: 0.5rem;
```

Comment:
this code is repeated in all 6 card dimensions. Please create a mixin that will set the width, the height and the margin:
```
@mixin set-card-dimensions($width: 20rem, $height: 20rem) {
     width: $width;
     height: $height:;
     margin: 0.5rem;
}
```

Then you will only have to pass values in the mixin when the width is not 20rem or the height is not 20rem.

```
.card-one-dimensions {
    @include set-card-dimensions(40rem, 16rem);
}

.card-three-dimensions {
    @include set-card-dimensions();
}
```

Reply:
created mixin used it.